ls1intum/Artemis
# [Exam] add connection status footer to participation view

### Checklist

### Motivation and Context


resolve #1607 
### Description

added a footer to the exam participation view to inform the students about their connection status.
Todo: implement the logic to save the work status in offline mode (not part of this PR and depends on the submission component)
### Steps for Testing


1. Navigate to courses/id/exam/id
2. Check that the footer style and information changes when you go offline/online

### Screenshots


![image](https://user-images.githubusercontent.com/50635005/84681184-1133f780-af34-11ea-97eb-d20f02ecc773.png)
![image](https://user-images.githubusercontent.com/50635005/84681358-4a6c6780-af34-11ea-85e1-343e60a1266d.png)

Updated sizing by @sascha11110 :

![image](https://user-images.githubusercontent.com/50635005/84709035-dba60300-af61-11ea-956b-f6c958053bfc.png)
![image](https://user-images.githubusercontent.com/50635005/84710259-245ebb80-af64-11ea-9235-4d30ba974a1f.png)
